project/ProjectDependencies.scala

```diff
@@ -22,6 +22,8 @@ object ProjectDependencyVersions {
   val Log4jVersion = "2.5"
   val ProtobufVersion = "2.5.0"
   val SparkVersion = "1.6.1"
+  val VertxVersion = "3.0.0"
+  val ExampleVertxVersion = "3.3.2"
```

Comment:
I choose different versions because the Adapter is compatible with Vert.x versions > 3.0.0, but version 3.0.0 has some logging issues and because of that the output in the examples was messed up.